variables.tf

```diff
@@ -158,3 +152,20 @@ variable "zone_id" {
   #
   # zone_id = "zoneid"
 }
+
+# ------------------------------------------------------------------------------
+# OPTIONAL MODULE CONFIGURATION PARAMETERS
```

Comment:
Hmm, I should have noted that in the last [PR](https://github.com/mineiros-io/terraform-module-template/commit/76b880ebaf229913da30c09a62a012313f5ef5bf). Not sure if I agree with adding another section. We already had a section for optional parameters https://github.com/mineiros-io/terraform-module-template/blob/master/variables.tf#L16

Reply:
This is from the template

Reply:
I get this, but why did you decide to split up the optional parameters into two sections anyways? Now we have three sections in the variables.tf. We already had a section prior to that. This seems like an unnecessary duplication to me. https://github.com/mineiros-io/terraform-module-template/blob/master/variables.tf#L17

Reply:
Those are pure module configuration not any optional arguments related to the actual service.. so to make this clear they got moved to the end to not care much.. ;)

Reply:
Ok, gotcha.